pkg/controller/podautoscaler/metrics/client.go

```diff
@@ -104,10 +101,10 @@ func getContainerMetrics(rawMetrics []metricsapi.PodMetrics, resource v1.Resourc
 			}
 		}
 		if !containerFound {
-			return nil, fmt.Errorf("container %s not present in metrics for pod %s/%s", container, m.Namespace, m.Name)
+			klog.FromContext(ctx).V(2).Info("Missing container metric", "container", container, "namespace", m.Namespace, "pod", m.Name)
```

Comment:
```suggestion
			klog.FromContext(ctx).V(2).Info("Missing container metric", "container", container, "pod", klog.Kobj(pod))
```

Reply:
We actually don't have the full `pod` resource here; just the `metricsapi.PodMetrics`. Updated it to use the same `"pod", klog.KRef(m.Namespace, m.Name)` that `getPodMetrics` is logging